activesupport/lib/active_support/core_ext/enumerable.rb

```diff
@@ -48,18 +48,29 @@ def maximum(key)
   # It can also calculate the sum without the use of a block.
   #
   #  [5, 15, 10].sum # => 30
-  #  ['foo', 'bar'].sum # => "foobar"
-  #  [[1, 2], [3, 1, 5]].sum # => [1, 2, 3, 1, 5]
+  #  ['foo', 'bar'].sum('') # => "foobar"
+  #  [[1, 2], [3, 1, 5]].sum([]) # => [1, 2, 3, 1, 5]
   #
   # The default sum of an empty list is zero. You can override this default:
   #
   #  [].sum(Payment.new(0)) { |i| i.amount } # => Payment.new(0)
   def sum(identity = nil, &block)
-    if identity
+    if identity 
```

Comment:
```suggestion
    if identity
```

Reply:
Thanks for taking the time to look at this and pointing that out - my formatter crashed w/out me noticing on the last push. Just updated the PR now.